vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/credentialverifier/statusChecker/LdpStatusChecker.kt

```
@@ -132,12 +140,12 @@ class LdpStatusChecker(verifierFactory: CredentialVerifierFactory = CredentialVe
             statusListVCMap = sendHTTPRequest(statusListCredentialUrl, GET)
                 ?: throw StatusCheckException(
                     "Failed to retrieve status list VC",
-                    StatusCheckErrorCode.STATUS_RETRIEVAL_ERROR
+                    STATUS_RETRIEVAL_ERROR
                 )
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)